history/cultures/alteraci.txt

```diff
@@ -0,0 +1,42 @@
+1.1.1 = {
```

Comment:
This is odd. Cultures don't have innovations that predict other ones.
The Alteraci has the Hereditary Rule, but doesn't have the Gavelkind.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/101753937-a0d5ef80-3aec-11eb-9878-ebb095c51e46.png)
![image](https://user-images.githubusercontent.com/46384992/101753999-b21efc00-3aec-11eb-979e-e089a8a6bbe2.png)

</details>
The Gilnean has the Bailiffs, but not the Ledger. Why would the Gilnean wait for the early medieval age to get +1 domain limit?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/101754384-1e016480-3aed-11eb-840c-f033130cba7c.png)
![image](https://user-images.githubusercontent.com/46384992/101754394-2194eb80-3aed-11eb-94f8-7f30945fb301.png)

</details>

Reply:
It wasn't fixed. All cultures have these problems with different techs. If you used the script, it definitely should be fixed there.
The Gilnean unlocked the Early Medieval military buildings without unlocking the Tribal military buildings.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/102485560-39d2b080-4081-11eb-8434-5212e81ff9e8.png)
![image](https://user-images.githubusercontent.com/46384992/102485846-9766fd00-4081-11eb-9f8f-f8b1e5e807b6.png)

</details>
It's like all cultures were rushing to the Early Medieval, spending more progress point while they could just discover the Tribal innovations.
Moreover, it's just not possible in normal games. There's a basic progress increase for every tech. So, if you had time to open 9 early medievals innovations, you already opened all the tribal innovations.